weblate/auth/models.py

```diff
@@ -613,7 +613,7 @@ class AutoGroup(models.Model):
         verbose_name=_("Regular expression for e-mail address"),
         max_length=200,
         default="^.*$",
-        help_text=_("Users with matching e-mail address will be added to this group."),
+        help_text=_("Users with a matching e-mail address will be added to this group."),
```

Comment:
```suggestion
        help_text=_("Users with an e-mail address found to match will be added to this group."),
```
Possible, but maybe more complicated?